.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Publish
+
+# Controls when the action will run. Triggers the workflow on push or pull request 
+# events but only for the master branch
+on:
```

Comment:
would stick to the painfully put together https://github.com/libero/reviewer-submission/blob/master/.github/workflows/ci.yml#L2-L7

Reply:
Ah thanks :+1: